scripts/shift_calculations.py

```diff
@@ -341,7 +341,7 @@ def plot_all_machines_in_year(
     color_map = {name: f"C{ii}" for ii, name in enumerate(["lhc", "sps", "ps", "psb", "leir", "ad", "superkekb"])}
 
     data_map: dict[str, float] = {}
-    for file_path in logbook_dir.glob(f"{year:4d}_*.md"):
+    for file_path in logbook_dir.glob(f"**/{year:4d}_*.md"):
```

Comment:
I notice `logbook_dir` is not passed to the function and not defined before. Is this an implicit global?

Reply:
It's not really a global, but within the scope of the function.... not really nice, but it's not an important script. can be refactored whenever

Reply:
Meh just wondering. It's not like the LHC runs on this script.